lib/node_modules/@stdlib/math/base/special/cceiln/README.md

```diff
@@ -95,38 +122,141 @@ var bool = ( v === out );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
+var uniform = require( '@stdlib/random/base/uniform' ).factory;
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' ).factory;
 var Complex128 = require( '@stdlib/complex/float64' );
-var randu = require( '@stdlib/random/base/randu' );
 var ceil = require( '@stdlib/math/base/special/ceil' );
-var real = require( '@stdlib/complex/real' );
-var imag = require( '@stdlib/complex/imag' );
 var cceiln = require( '@stdlib/math/base/special/cceiln' );
 
-var re;
-var im;
+var rand = uniform( -50.0, 50.0 );
+var randn = discreteUniform( -5.0, 0.0 );
```

Comment:
```suggestion
var rand2 = discreteUniform( -5.0, 0.0 );
```

Reply:
`randn` is a commonly used alias for a PRNG generating normally distributed numbers.